### ISSUE TITLE
Collections in the strapi admin panel are empty

### ISSUE THREAD

**Issue by nphardorworse** [meilisearch/strapi-plugin-meilisearch#661]
**Description**
Hi, I was happily using the plugin for a couple of weeks and today I've noticed that the collections are no longer loaded in the strapi admin panel. Not sure when it first became an issue. I don't get any error logs anywhere. Separately, both strapi and meilisearch are working as expected. I created a new meilisearch instance to test, but it still didn't work. I have no hint of what might be wrong. I'd greatly appreciate any help in figuring it out or directing me to some common issues perhaps. Thanks. 

**Environment:**
- OS: macOS 
- Meilisearch version: 1.0.1
- strapi-plugin-meilisearch version: 0.9.1
- Browser: Safari 16.1/Brave v1.48.164
- Strapi version: 4.6.2 (but also didn't work with 4.6.1)

**Comment by bidoubiwa** [comment]
Hey @nphardorworse 

I tried locally with strapi v4.6 and it seems to work well. So I don't think it comes from an incompatibility between the versions.
Can you run your strapi app in dev mode and watch both browser console and server console outputs when you enter the meilisearch plugin page?

What database are you using?

**Comment by nphardorworse** [comment]
It seems like the db was somehow the culprit. I rebooted my db instance (aws RDS) and now the collections are back. I don't understand what exactly went wrong or how to prevent it or even detect it in the future, but well, it's working.

Just to emphasize again: I did not get any error message anywhere, strapi was working without any issues, I could connect to the database by other means as well. I tried now to look into the db log to see if I could find anything insightful there, but I found nothing obvious (at least for me). 

Thanks again for your help @bidoubiwa
